modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/resources/content/Language.properties

```diff
@@ -1,55 +1,57 @@
+network-publish-host-help=This configuration only applies to EMBEDDED (Development) mode. Set the single interface the node advertises to other nodes in the cluster, so that those nodes can connect to it.
 node-name=Node Name
-node-name-help=This configuration only applies to EMBEDDED mode. Set the name of the node to start. If unset, "liferay" is used.
+node-name-help=This configuration only applies to EMBEDDED (Development) mode. Set the name of the node to start. If unset, "liferay" is used.
 number-of-index-replicas=Number of Index Replicas
 number-of-index-replicas-help=Set the number of replicas for each Liferay index. If unset, no replicas are used. Changes to this value take effect after a full reindex.
 number-of-index-shards=Number of Index Shards
 number-of-index-shards-help=Set the number of shards to use when a Liferay index is created. If unset, a single shard will be used. Changes to this value take effect after a full reindex.
 operation-mode=Operation Mode
-operation-mode-help=The REMOTE operation mode connects to a remote standalone Elasticsearch cluster. The EMBEDDED operation mode starts Portal with an internal Elasticsearch instance in the same JVM. The EMBEDDED operation mode is unsupported.
+operation-mode-help=This configuration is deprecated. The REMOTE (Production) operation mode connects to a remote standalone Elasticsearch cluster, which can also be achieved by enabling Production mode. The EMBEDDED (Development) operation mode starts a child process running an external sidecar Elasticsearch instance. The EMBEDDED (Development) operation mode is unsupported. If Production Mode is enabled, this setting will be ignored.
 override-type-mappings=Override Type Mappings
 override-type-mappings-help=Set custom mappings to fully replace the default mappings for the LiferayDocumentType, in JSON format (Elasticsearch Put Mapping API). If you set this value, index creation ignores the default mappings definition and any extension points, using this alternative mappings definition instead. Use this if you need LiferayDocumentType to be created with nonstandard mappings for default fields. If you just need to add new custom mappings without overlapping any defaults, use additionalTypeMappings.
 password-help=Set the password for authenticating to Elasticsearch if Authentication Enabled is checked.
+production-mode-enabled=Production Mode Enabled
+production-mode-enabled-help=Enable or disable Production mode. If this is checked, Production mode will be enabled and the Operation Mode configuration will be ignored. Production mode connects to a remote standalone Elasticsearch cluster. If this is unchecked, Development mode will be enabled. Development mode starts a child process running an external sidecar Elasticsearch instance. Development mode is unsupported. For backwards compatibility, if this is unchecked, the Operation Mode configuration will be used instead.
```

Comment:
> production-mode-enabled-help=Enable or disable Production mode. If this is checked, Production mode will be enabled and the Operation Mode configuration will be ignored. Production mode connects to a remote standalone Elasticsearch cluster. If this is unchecked, Development mode will be enabled. Development mode starts a child process running an external sidecar Elasticsearch instance. Development mode is unsupported. For backwards compatibility, if this is unchecked, the Operation Mode configuration will be used instead.

I find the _unchecked_ sentences confusing. I can't tell if both can be perfectly true.

> If this is unchecked, Development mode will be enabled.

but what if you set REMOTE in operation mode?

> For backwards compatibility, if this is unchecked, the Operation Mode configuration will be used instead.

Reply:
i know... so i what i was trying to do was write the final version of the key ("If this is unchecked, Development mode will be enabled.") and then tack on the explanation of the current behavior ("For backwards compatibility, if this is unchecked, the Operation Mode configuration will be used instead") at the end so that it can be easily deleted in future versions. but maybe i should just remove the "If this is unchecked, Development mode will be enabled." sentence for now?

Reply:
and to answer your question, if you uncheck the box and set to operation mode to REMOTE, its gonna be REMOTE

Reply:
how about this? simplified as much as i can
> Enable Production mode. If this is checked, Production mode will be enabled and the Operation Mode configuration will be ignored. Production mode connects to a remote standalone Elasticsearch cluster. If this is unchecked, the Operation Mode configuration will be used.

Reply:
Ahh, now I understand what you were doing.

I like the version you suggested above.